### PR TITLE
Kamu UI 434 ux resizable query input

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**
Closes: https://github.com/kamu-data/kamu-web-ui/issues/434

Result:

image

### CODE REVIEW

File: src/app/shared/editor/components/base-editor/base-editor.component.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { EditorError } from "../../models/error.model";
 export abstract class BaseEditorComponent implements OnChanges {
     @Input({ required: true }) public template = "";
     @Input() public error: MaybeNull<string>;
-    @Input() public height: MaybeUndefined<string>;
+    @Input() public height: MaybeUndefined<string> = "220px";
```
Comment (zaychenko-sergei) [inline src/app/shared/editor/components/base-editor/base-editor.component.ts:13]: Do we need undefined if we have a default?

File: src/app/dataset-view/additional-components/data-component/data.component.ts [hunk 2]
```diff
@@ -64,6 +64,63 @@ export class DataComponent extends BaseComponent implements OnInit {
...
+        //     fromEvent(document, "touchstart").pipe(
+        //         map((event) => ({
+        //             target: event.target,
+        //             screenX: (event as TouchEvent).touches[0].screenX,
+        //             screenY: (event as TouchEvent).touches[0].screenY,
+        //         })),
+        //     ),
+        // )
+        //     .pipe(
+        //         filter((event) => {
```
Comment (zaychenko-sergei) [inline src/app/dataset-view/additional-components/data-component/data.component.ts:78]: Garbage

File: src/app/shared/editor/components/base-editor/base-editor.component.ts (hunk 1 above)
Comment (dmitriy-borzenko) [reply src/app/shared/editor/components/base-editor/base-editor.component.ts:13]: Yes, it is unnecessary

File: src/app/dataset-view/additional-components/data-component/data.component.ts (hunk 2 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/additional-components/data-component/data.component.ts:78]: Removed

**Review by zaychenko-sergei — APPROVED** [review]

**Pull request merged**